### ISSUE TITLE
Build fails to start

### ISSUE THREAD

**Issue by XCVG** [dieno/GarlicGang#1]
UnityPlayer.dll is broken and the game fails to start

**Comment by XCVG** [comment]
All DLLs seem to be broken with very small file sizes. Possibly an issue with LFS.

**Comment by dieno** [comment]
Seems to work if you pull from the repo. Looks like Git LFS files aren't included if you download the zip from github. Are you pulling the repo?

**Comment by XCVG** [comment]
I downloaded rather than pulling. I think most people will do the same, so that's not really a fix.

**Comment by dieno** [comment]
Yeah, good point. I'll throw it up on Itch.io because I have no idea about LFS. It will probably be better that way. Unless you have any better suggestions.
